portfolio/src/main/webapp/script.js

```diff
@@ -32,7 +32,7 @@ function addRandomFact() {
 }
 
 function sayHello() {
-    fetch("/data").then(response => response.text()).then((text) => {
+    fetch("/data").then(response => response.json()).then((text) => {
     document.getElementById('hello-container').innerText = text;
```

Comment:
It looks like you are putting the entire list of greetings in the innerText here. I'm guessing that isn't what you wanted to do.

Reply:
Yeah, Marti. I was a little confused about that too. I spoke with someone from the VSE chat about it and they said that it was okay, but I agree. I think I fixed with it in my new commits, let me know!